kratos/input_output/obj_io.cpp

```diff
@@ -129,8 +129,18 @@ void ObjIO::ReadModelPart(ModelPart& rThisModelPart)
     // Clean up problematic geometries
     const bool clean_up_problematic_geometries = mParameters["clean_up_problematic_geometries"].GetBool();
     if (clean_up_problematic_geometries) {
-        const double area_tolerance = mParameters["area_tolerance"].GetDouble();
-        CleanUpProblematicGeometriesInMesh(rThisModelPart, entity_type, mFirstNodeId, mFirstElementId, mFirstConditionId, area_tolerance);
+        Parameters clean_up_parameters = Parameters(R"({
+            "entity_type"        : "element",
+            "first_node_id"      : 1,
+            "first_element_id"   : 1,
+            "first_condition_id" : 1
+        })");
+        clean_up_parameters["entity_type"].SetString(entity_type);
+        clean_up_parameters["first_node_id"].SetInt(mFirstNodeId);
+        clean_up_parameters["first_element_id"].SetInt(mFirstElementId);
+        clean_up_parameters["first_condition_id"].SetInt(mFirstConditionId);
+        clean_up_parameters.AddValue("area_tolerance", mParameters["area_tolerance"]);
+        CleanUpProblematicTrianglesModeler(rThisModelPart, clean_up_parameters).SetupModelPart();
```

Comment:
I'm not in love with the idea of silently composing the modelers one inside the other. On top of being more transparent, wouldn't it be easier to just call one after the other in the json? Note that this is would be consistent to what we do with other I/O options.

Reply:
Yes, but this was already in the IO, I jst refactored as a modeler. If I take it it would break behaviour consistency.

Reply:
Well, `obj_io` was added less than one month ago. I don't think there's any problem on breaking current behavior. Nevertheless, if this is a blocker I'd leave the option and throw a deprecation error to warn current users.

Reply:
Well, as you are TC, I will accept. I was trying to "follow the rules".